melos.yaml

```
@@ -22,3 +22,6 @@ scripts:
   format-check: dart format --output=none --set-exit-if-changed --line-length 120 .
   analyze: dart analyze --fatal-infos .
   test: melos exec --concurrency=1 --fail-fast --dir-exists=test -- "flutter test --concurrency=$(nproc --all)"
+  generate:neon:build_runner: melos exec --scope="neon*" --file-exists="build.yaml" -- dart run build_runner build --delete-conflicting-outputs && melos run format
+  format: dart format --fix --line-length 120 .
+  generate:l10n: melos exec --flutter --dir-exists="lib/l10n" flutter gen-l10n && melos run format
```

```suggestion
  generate:neon:l10n: melos exec --flutter --dir-exists="lib/l10n" flutter gen-l10n && melos run format
```

Could also do this since it's the only use case, but technically not limited so can leave it as well